third_party/protobuf/src/google/protobuf/stubs/atomicops_internals_generic_gcc.h

```diff
@@ -128,6 +128,24 @@ inline Atomic64 NoBarrier_CompareAndSwap(volatile Atomic64* ptr,
   return old_value;
 }
 
+inline Atomic64 NoBarrier_AtomicIncrement(volatile Atomic64* ptr,
```

Comment:
@dslomov  The above change has been committed in protobuf master through commit id [c59473d](https://github.com/google/protobuf/commit/c59473d53eafadd126502657e5c5c33e952b67ed) 
However, it looks Bazel doesn't use protobuf master yet.
Hence, I have made changes in 
`bazel/third_party/protobuf/src/google/protobuf/stubs/atomicops_internals_generic_gcc.h`